pom.xml

```diff
@@ -57,8 +58,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.387.x</artifactId>
-        <version>2143.ve4c3c9ec790a</version>
+        <artifactId>bom-2.414.x</artifactId>
```

Comment:
@MarkEWaite I guess you will create `2.426.x` some time soon.

Reply:
Yes, I hope to stay the 2.426.x checklist by the end of the week